### PR TITLE
Code Refactoring and Formatting

### PR DESCRIPTION
**Pull request by Kamidu**
This PR included Fix for the Jira Issue: EMM-1242.

### CODE REVIEW

File: modules/apps/jaggery/emm/src/emm/modules/user.js [hunk 1]
```diff
@@ -65,39 +65,27 @@ var userModule = function () {
...
-                        var response = {};
-                        response.content = responsePayload;
-                        response.status = "error";
-                        return response;
-                    });
+        if (constants.HTTP_GET == method) {
+            var response = serviceInvokers.XMLHttp.get(url, function (responsePayload) {
+                var response = {};
+                response.content = responsePayload["responseContent"];
+                if(responsePayload["responseContent"] == null && responsePayload != null){
```
Comment (prabathabey) [inline modules/apps/jaggery/emm/src/emm/modules/user.js]: Formatting issues.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/home/MDMHomePage.java [hunk 2]
```diff
@@ -28,8 +28,10 @@
 
 import java.io.IOException;
 
+/**
+ * class represent MDM home page and its behaviors.
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/home/MDMHomePage.java]: Wrong grammar in the comment.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java [hunk 3]
```diff
@@ -41,32 +45,41 @@ public PlatformConfiguration(WebDriver driver) throws IOException {
         }
     }
 
-    public void changeGeneralConfig(String value){
+    /**
+     * This page imitate the general configuration changing scenario.
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Please fix wrong grammar used in the comment.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java [hunk 4]
```diff
@@ -41,32 +45,41 @@ public PlatformConfiguration(WebDriver driver) throws IOException {
...
         WebElement buttonSaveConfig = driver.findElement(By.xpath((uiElementMapper.getElement("emm.configuration.general.button.save.identifier"))));
-
         generalConfigButton.click();
         inputGeneralConfig.sendKeys(value);
         buttonSaveConfig.click();
     }
 
-    public void changeAndroidConfig(String value){
+    /**
+     * This page imitate the android configuration changing scenario.
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Please fix wrong grammar used in the comment.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java [hunk 5]
```diff
@@ -41,32 +45,41 @@ public PlatformConfiguration(WebDriver driver) throws IOException {
...
-
         androidConfigButton.click();
         inputAndroidConfig.sendKeys(value);
         buttonSaveConfig.click();
 
     }
 
-    public void changeWindowsConfig(String value){
+    /**
+     * This page imitate the windows configuration changing scenario.
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Please fix wrong grammar used in the comment.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java [hunk 6]
```diff
@@ -27,11 +43,19 @@
...
+/**
+ * Class to represents Notification View.
+ */
 public class NotificationView {
     private WebDriver driver;
     private UIElementMapper uiElementMapper;
     private static final Log log = LogFactory.getLog(NotificationView.class);
 
+    /**
+     * Constructor
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java]: This comment is redundant I believe.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java [hunk 7]
```diff
@@ -27,11 +43,19 @@
 
 import java.io.IOException;
 
+/**
+ * Class to represents Notification View.
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java]: Please fix wrong grammar used in the comment.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java (hunk 5 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java (hunk 4 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java (hunk 3 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/home/MDMHomePage.java (hunk 2 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/home/MDMHomePage.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java (hunk 6 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java (hunk 7 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/Notification/NotificationView.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/apps/jaggery/emm/src/emm/modules/user.js (hunk 1 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/modules/user.js]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/apps/jaggery/emm/src/emm/units/device-detail/public/js/device-detail.js [hunk 8]
```diff
@@ -182,7 +185,8 @@
                 viewModel.policy = activePolicy;
                 viewModel.deviceType = deviceType;
                 data = JSON.parse(data);
-                if(data != null && data.complianceFeatures!= null && data.complianceFeatures != undefined && data.complianceFeatures.length > 0) {
+                if(data != null && data.complianceFeatures!= null && data.complianceFeatures != undefined &&
```
Comment (prabathabey) [inline modules/apps/jaggery/emm/src/emm/units/device-detail/public/js/device-detail.js]: Formatting issues.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java [hunk 9]
```diff
@@ -24,8 +24,12 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.wso2.emm.integration.ui.pages.UIElementMapper;
+
 import java.io.IOException;
 
+/**
+ * Class to represent the Platform Configuration Page.
```
Comment (prabathabey) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: This comment doesn't seem to align with the pattern followed in other similar test cases.

File: modules/apps/jaggery/emm/src/emm/units/device-detail/public/js/device-detail.js (hunk 8 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/device-detail/public/js/device-detail.js]: Duly Noted.Suggested changes are made and PR is updated accordingly.

File: modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java (hunk 9 above)
Comment (Kamidu) [inline modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java]: Duly Noted.Suggested changes are made and PR is updated accordingly.

**Pull request merged**